rockcraft.yaml

```diff
@@ -0,0 +1,170 @@
+---
+name: sdcore-upf-bess
+base: ubuntu:20.04
+version: '1.3'
+summary: SD-Core UPF BESS
+description: SD-Core UPF BESS
+license: Apache-2.0
+platforms:
+  amd64:
+
+parts:
+  xdp:
+    plugin: autotools
+    source: https://github.com/xdp-project/xdp-tools.git
+    source-tag: v1.2.2
+    build-packages:
+      - clang
+      - gcc-multilib
+      - libelf-dev
+      - libpcap-dev
+      - linux-headers-5.4.0-26
+      - llvm
+    prime:
+      - usr/local/lib/x86_64-linux-gnu/*.so
+      - usr/lib/libxdp*
+
+  cndp:
+    after:
+      - xdp
+    plugin: nil
+    source: https://github.com/CloudNativeDataPlane/cndp.git
+    source-commit: d5ce4b9edc2e7ddb46a61b395deffafaf11a0500
+    build-environment:
+      - CNE_DEST_DIR: "${CRAFT_PART_INSTALL}"
+    build-packages:
+      - clang
+      - golang
+      - libbpf-dev
+      - libbsd-dev
+      - libnl-3-dev
+      - libnl-cli-3-dev
+      - libnuma-dev
+      - lld
+    override-build: |
+      make
+      make install
+      make static_build=1 rebuild install
+      pushd lang/go/stats/prometheus/
+      go build -o $CRAFT_PART_INSTALL/prometheus prometheus.go
```

Comment:
Why are building Prometheus here? It feels like we could leverage a plugin (make) to build `cncp` here and have the prometheus client build in a different part.

Reply:
I am not sure this binary is required, I will test without it.

Reply:
I have removed the prometheus portion and used the make plugin